test/e2e-go/features/participation/onlineOfflineParticipation_test.go

```diff
@@ -159,35 +174,45 @@ func TestNewAccountCanGoOnlineAndParticipate(t *testing.T) {
 	newAccountStatus, err := client.AccountInformation(newAccount)
 	a.NoError(err, "client should be able to get information about new account")
 	a.Equal(basics.Online.String(), newAccountStatus.Status, "new account should be online")
+
+	nodeStatus, _ = client.Status()
+	fundedRoundBefore := nodeStatus.LastRound
+	
 	// account receives almost all of rich account's stake (minus enough to
 	// keep it over MinBalance), so it will be selected for participation
 	amountToSend := richBalance - 3*transactionFee - amountToSendInitial - minAcctBalance
 	fixture.SendMoneyAndWait(onlineRound, amountToSend, transactionFee, richAccount, newAccount)
```

Comment:
Since you're sending all the money, I would advise to sent it as a closeto, so that the old account would be removed from the system. This would make sure the test is more robust.

Reply:
Done.